dask/array/utils.py

```diff
@@ -273,9 +274,12 @@ def assert_eq(
     check_chunks=True,
     check_type=True,
     equal_nan=True,
-    scheduler="sync",
+    scheduler=None,
     **kwargs,
 ):
+    if scheduler is None:
+        scheduler = config.get("testing.assert-eq.scheduler")
```

Comment:
Note that we would be using the old style of `config.get`, as I don't think we want Dask using the single-threaded scheduler by default:

```suggestion
        scheduler = config.get("scheduler", "sync")
```